server/src/dynamodb.ts

```diff
@@ -89,40 +68,27 @@ export class Dynamo implements Database {
       ReturnValues: 'UPDATED_NEW',
     };
 
-    try {
-      const resp = await this.dynamoDb.update(params).promise();
-      console.log('Successfully updated item', JSON.stringify(resp));
-      return projectId;
-    } catch (err) {
-      console.log(`Unable to update listoProject with id ${projectId}`, err);
-      throw err;
-    }
+    const resp = await this.db.update(params).promise();
+    console.log('Successfully updated item', JSON.stringify(resp));
+    return projectId;
   }
 
-  public async getProject(projectId: string): Promise<DatabaseModel> {
-    try {
-      const params = {
-        TableName: tableName,
-        Key: {
-          id: projectId,
-        },
-      };
+  public async find(projectId: string): Promise<DatabaseModel> {
+    const params = {
+      TableName: tableName,
+      Key: {
+        id: projectId,
+      },
+    };
 
-      const data = await this.dynamoDb.get(params).promise();
+    const data = await this.db.get(params).promise();
 
-      if (!Object.keys(data).length) {
-        throw 'Project not found';
-      }
+    if (!Object.keys(data).length) {
```

Comment:
Need to be careful here since `Object.keys` will throw an error if `data` is `null` or `undefined`... especially since you removed the `try .. catch`

Reply:
```suggestion
    if (!data.Item) {
```

Per https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB/DocumentClient.html#get-property `data` will be null on an error, but if the item isn't found it's only `Item` that will be missing as the lookup will still consume resources and thus these stats must be returned to you.